src/action/interactive.rs

```diff
@@ -196,7 +196,9 @@ impl UserPicked {
                 return Ok(Pick::Replacement(bandaid));
             }
             KeyCode::Esc => return Ok(Pick::Quit),
-            KeyCode::Char('c') if modifiers == KeyModifiers::CONTROL => return Ok(Pick::Quit),
+            KeyCode::Char('c') if modifiers == KeyModifiers::CONTROL => {
+                return Err(anyhow!("Received SIGINT"))
```

Comment:
will cause an error code `1`, and not `130` - I think we should exit the process right there

Reply:
I wasn't sure if it's a good idea to exit right there.